addons/events/fnc_addPlayerEventHandler.sqf

```diff
@@ -189,7 +192,24 @@ if (_id != -1) then {
                 [QGVAR(turretEvent), [_player, _data]] call CBA_fnc_localEvent;
             };
 
-            _data = currentVisionMode _player;
+            // handle controlling UAV, UAV entity needed for visionMode
+            _data = UAVControl getConnectedUAV _player;
+            if !(_data isEqualTo GVAR(oldUAVControl)) then {
+                GVAR(oldUAVControl) = _data;
+
+                private _role = _data param [(_data find _player) + 1, ""];
```

Comment:
Is the wiki page wrong?
https://community.bistudio.com/wiki/UAVControl
_data should be 2 element array. find _player should always return 0.

Reply:
>Is the wiki page wrong?

Always.

Drone with turret + driver/pilot looks like this:
[object1, "DRIVER", object2, "GUNNER"]
In MP, it can be controlled by multiple people. Therefore #0 is not necessarily the local player.